src/panel/direction/RouteResult.jsx

```diff
@@ -13,8 +13,7 @@ export default class RouteResult extends React.Component {
     destination: PropTypes.string,
     vehicle: PropTypes.string,
     isLoading: PropTypes.bool,
-    error: PropTypes.bool,
-    errorCode: PropTypes.number,
+    error: PropTypes.number,
```

Comment:
This is correct but now you need to change the initial `error` state of `DirectionPanel` from `false` to `0` to avoid this warning: 

![Capture d’écran de 2020-02-11 16-09-28](https://user-images.githubusercontent.com/243653/74249105-ebc68100-4ce8-11ea-9a11-9b314528b1a9.png)